quartz/src/main/java/org/quartz/impl/triggers/DailyTimeIntervalTriggerImpl.java

```diff
@@ -875,8 +875,6 @@ public Set<Integer> getDaysOfWeek() {
     public void setDaysOfWeek(Set<Integer> daysOfWeek) {
         if(daysOfWeek == null || daysOfWeek.isEmpty())
             throw new IllegalArgumentException("DaysOfWeek set must be a set that contains at least one day.");
-        else if(daysOfWeek.isEmpty())
```

Comment:
Do we want to fix this by changing the previous condition and modifying the messages?

Reply:
for me it is redundant both messages. The fact would be to choose which one will be left :). There is a pr opened for this :) https://github.com/quartz-scheduler/quartz/issues/1209